OfficeAddin/05 Using modern JavaScript/Demos/01 Office Add-in using React/config/webpack.dev.js

```diff
@@ -1,36 +1,29 @@
-const path = require('path');
-const webpack = require('webpack');
-const webpackMerge = require('webpack-merge');
-const commonConfig = require('./webpack.common.js');
-const BrowserSyncPlugin = require('browser-sync-webpack-plugin');
-
-module.exports = webpackMerge(commonConfig, {
-    devtool: 'eval-source-map',
-
-    plugins: [
-        new BrowserSyncPlugin(
-            {
-                https: true,
-                host: 'localhost',
-                port: 3000,
-                proxy: 'https://localhost:3100/'
-            },
-            {
-                reload: false
-            }
-        )
-    ],
-
-    devServer: {
-        publicPath: '/',
-        contentBase: path.resolve('dist'),
-        https: true,
-        compress: true,
-        overlay: {
-            warnings: false,
-            errors: true
-        },
-        port: 3100,
-        historyApiFallback: true
-    }
-});
+const fs = require('fs');

```

Comment:
missing copyright and license header

Reply:
? I just created a brand new project I don't see any license / copyright comments in any of the `webpack.*.js` generated files